github_app.go

```diff
@@ -0,0 +1,231 @@
+package evergreen
```

Comment:
this shouldn't be top-level; this should be under model, same as the deleted files.

Reply:
Can you add a comment to that ticket? And maybe assign to your next sprint, I think we should do this ASAP so this doesn't get any worse